src/config.py

```diff
@@ -1,10 +1,42 @@
 import os
+import psycopg2
+from sqlalchemy import create_engine
 from dotenv import load_dotenv
+from src.helper_functions import get_logger
 
 load_dotenv()
 ETHEREUM_NODE_URL = os.getenv("ETHEREUM_NODE_URL")
 GNOSIS_NODE_URL = os.getenv("GNOSIS_NODE_URL")
 
+ARBITRUM_NODE_URL = os.getenv("ARBITRUM_NODE_URL")
+SOLVER_SLIPPAGE_DB_URL = os.getenv("SOLVER_SLIPPAGE_DB_URL")
+
 CHAIN_RPC_ENDPOINTS = {"Ethereum": ETHEREUM_NODE_URL, "Gnosis": GNOSIS_NODE_URL}
 
-CHAIN_SLEEP_TIMES = {"Ethereum": 60, "Gnosis": 120}
+# sleep time can be configured here
+CHAIN_SLEEP_TIMES = {"Ethereum": 60.0, "Gnosis": 120.0}
```

Comment:
Should we have the sleep times as environment variables so that we can update them more easily?

Reply:
yeah, makes sense